apps/web/src/components/Basenames/RegistrationSearchInput/index.tsx

```diff
@@ -304,9 +318,13 @@ export default function RegistrationSearchInput({
           </p>
         ) : (
           <>
-            <p className={mutedMessage}>
-              {formatBaseEthDomain(debouncedSearch, basenameChain.id)} is already registered
-            </p>
+            <button className={buttonClasses} type="button" onClick={viewBasenameProfile}>
```

Comment:
This should be a Link from

```
apps/web/src/components/Link/index.tsx
```

so we can track analytics and remove router usage.